README.md

```diff
@@ -1,21 +1,94 @@
+ 
+*Are you super-advanced?* Do you want to build your own tools? We 💖 contributions. 
+
+## Design principles
+
+*Classroom is a teacher-facing tool to enable the educational use of GitHub.* Every student needs feedback and support as they learn to code, and using GitHub you can give them the right advice, in the right place, at the right time. This tool makes it easier for you to use the workflow you love, but with some organization, automation and ease with your students. 
+ 
+*Students use GitHub. They don’t use Classroom.* Experience with real-world tools gives students a leg-up once they move on from school. Invest time in the tools students can grow with, not another third-party tool with its own learning curve. 
+ 
+*Classroom is not an LMS (Learning Management System).* If you need to use an LMS like Canvas, Moodle or Blackboard, we hear you. Students can post their repositories to your LMS. We’re going to stick with what we’re good at, which is helping people collaborate on code. 
+ 
+*Classroom is open source.* Git and GitHub are versatile with many ways to meet your learning goals, and we want to model the open source process that makes our communities great. 
+
+We welcome contributions aligned with the roadmap below and through answering these questions: 
+ 
+* Does it enable the real-life experience of using GitHub as a developer? 
+* Does it replicate functionality of apps, hardware or LMS? 
+* How many support tickets and questions will the feature address? 
+ 
+### Who is Classroom for?
+Anyone teaching computer science in a high school, university or informal environment.
+Folks who might also find Classroom useful:
+* Higher ed statistics and data science teachers
+* Higher ed biology and the hard sciences
+
+## GitHub Classroom and the edtech ecosystem
+
+In case you’re wondering “How does Classroom interact with my favorite app/my notebook/my LMS” here’s the tl;dr on how those pieces fit together: 
+ 
+*Apps and content platforms*
+Examples: Codecademy, Skillshare, Udemy, Udacity
+Apps offer premium content and interactive exercises. GitHub Classroom offers *real-world experience* with code. Classroom, as a teacher-facing application will eventually surface best-in-class content for top courses (notes / lectures / problem sets) but not produce original content. 
+ 
+*Learning Management system/LMS*
+Examples: Blackboard, Moodle, Canvas. Google Classroom
+Teachers often use a learning management system in keeping with student privacy regulations. Classroom has a lightweight integration with LMS ecosystem--students can submit a link to their repositories. LTI compliance and Google Classroom integration will make the app more extensible. 
+ 
+*Notebooks* 
+Examples: BlueJ, Jupyter, RStudio
+Most notebooks have a Git integration that students can push to. Future iterations may pre-populate repos with directions on set up.
```

Comment:
We can already initialize repo's with starter-code, it's just super basic right now